src/spring/azext_spring/app.py

```diff
@@ -376,9 +378,39 @@ def app_deploy(cmd, client, resource_group, service, name,
                    'while to complete)'.format(kwargs['total_steps'],
                                                kwargs['total_steps'],
                                                name))
-    return sdk_no_wait(no_wait, deployment_factory.get_deploy_method(**kwargs),
-                       resource_group, service, name, deployment.name,
-                       deployment_resource)
+    poller = sdk_no_wait(no_wait, deployment_factory.get_deploy_method(**kwargs),
```

Comment:
any parameter to let customer decide to show these application log? Otherwise the existing script will be broke